src/main.js

```diff
@@ -0,0 +1,32 @@
+var server = require('http').createServer();
+var io = require('socket.io')(server);
+import {Monopoly} from "./monopoly.js";
+import {PlayerList} from "./player.js";
+
+
+
+
+io.on('connect', function(socket){
+  //assign user
+  socket.on('getAccountVal', function(playerId){
+    io.emit('getAccountVal', Monopoly.getAccountVal(playerId));
+  });
+
+  socket.on('rollDice', function(){
+    dice = Monopoly.rollDice();//should also make monopoly member utilize roll action (e.g. move player on roll)
```

Comment:
Always declare your variables with `let` or `const`.  If you don't your variables get declared globally and can lead to nasty bugs and memory inefficiency.  

The bugs occur when you name a variable that matches something already in the global object and overwrite it.  Memory wise your variables are typically marked for clean up at the end of their scope (I.e. the function they were made in. ). But if defined globally they're only cleaned up if you do it explicitly or when the program closes.